packages/ckeditor5-ui/src/toolbar/toolbarview.js

```diff
@@ -277,6 +288,22 @@ export default class ToolbarView extends View {
 		this.items.addMany( config.map( name => {
 			if ( name == '|' ) {
 				return new ToolbarSeparatorView();
+			} else if ( name == '-' ) {
+				if ( this.options.shouldGroupWhenFull ) {
+					/**
+					 * Toolbar line break will work only when button grouping is disabled in the toolbar config:
+					 *
+					 * 		Editor.defaultConfig = {
+					 * 			toolbar: {
+					 * 				shouldNotGroupWhenFull: true
+					 * 			}
+					 * 		}
+					 *
+					 * @error line-separator-used-when-button-grouping-enabled
+					 */
+					logWarning( 'line-separator-used-when-button-grouping-enabled' );
```

Comment:
Maybe let's log `config` along with this error. There could be multiple editors with different toolbar configs on the same page and for this error to be useful we should give developers some context.

Reply:
done